CHANGES.rst

```diff
@@ -42,6 +42,8 @@ Specviz2d
 
 - Spectral extraction plugin. [#1514, #1560, #1562]
 
+- CLI support for launching Specviz2d (2D spectrum only). [#1576]
```

Comment:
I don't understand the emphasize on "2D spectrum only" if I were a user and reading this entry without further insider knowledge. Please explain more.

Reply:
The CLI which is shared across all configs only accepts a single path (filename or directory).  The specviz2d parser currently takes either 1 filename (2d spectrum, automatically extracting the 1d spectrum) or 2 filenames (2d spectrum and 1d spectrum).  Supporting that will either take significant work to the parser, or an additional argument in the CLI just for specviz2d - as explained in the PR description.  I think that decision will require discussion but shouldn't necessarily block the basic support introduced here.  I'm open to suggestions for making that more clear in the changelog though without getting too technical!

Reply:
```suggestion
- CLI support for launching Specviz2d for a single 2D spectrum file input.
  Use notebook version if you want to open separate 2D and 1D spectra in Specviz2d. [#1576]
```